src/neo/SmartContract/Native/RoleDesignation.cs

```diff
@@ -14,12 +12,12 @@
 
 namespace Neo.SmartContract.Native
 {
-    public sealed class DesignationContract : NativeContract
+    public sealed class RoleDesignation : NativeContract
```

Comment:
Maybe keep with `ContractRoleDesignation` to follow `ContractManagement`

Reply:
I don't understand.

Reply:
Now it is `RoleManagement`.